upup/pkg/fi/cloudup/nodeterminationhandlertasks/sqs.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeterminationhandlertasks
```

Comment:
Not sure we really need a dedicated package for this. In the future it may also be other reasons for kOps to provision an SQS.

Reply:
sounds good, I'll move them out to `awstasks`